playground-next13-live-editing/components/Teaser.tsx

```diff
@@ -0,0 +1,12 @@
+// "use server";
+import { storyblokEditable } from "@storyblok/react/rsc";
+
+const Teaser = ({ blok }) => {
+  return (
+    <h2 className="teaser" {...storyblokEditable(blok)}>
```

Comment:
The class teaser is not used.

Reply:
Oh I see, the best way to handle for testing would be : `data-cy=teaser` for exemple  or `data-test`/`data-testid`
https://docs.cypress.io/guides/references/best-practices